### DISCUSSION TITLE
🚓 Увеличить время пребывания машины без водителя у DPD и WCSO

### DISCUSSION THREAD

**Discussion by damirmd** [octothorp-team/public-dobrograd#846] [Предложения]
**Увеличить время пребывания машины без водителя у ДПД и WCSO – не нагружает сервер, но удобно!**

> Зачем это нужно?

На ситуации иногда нужно взять локерное или бронежилет. Но как на зло машина автоматически эвакуируется! Такая система у всех машин на сервере, но для полицейских сообществ это неудобно. Приходится оформлять снова машину и бежать с другого конца города в гараж, или на крайний случай брать снаряжение у других крузеров. Так вот предлагаю увеличить время нахождения полицейских машин без водителя. Нагрузка не изменится, но это будет максимально удобно для коммьюнити!

> Примечание

_Обычным муниципалам советую не добавлять это._

**Comment by KillerBunny031** [comment]
Блять почему к муниципалам относятся как к скоту? Почему у муниципалов есть только стандартный набор тен кодов? Почему все идея для улучшения полиции никогда не относятся к муниципалам? Когда заходят ДПДшники и начинают говорить в рацию, муниципалу вообще ничего не понять. Они как будто на другом языке говорят. Давайте перестанем так относиться к муникам. У миниципалов тоже часто пропадает машина в самый неподходящий момент. Но ЧСВшники в ДПД как всегда думают только о себе.

**Reply by damirmd** [reply]
Опять же разводить срач не хочу.
 Это предложение чтобы можно было брать локер или вернуть его назад без проблем. Муниципалам это не нужно по причине всего оружия в баре. Да, им нужно отыгрывать также, но по механу оружие у них есть.

**Reply by damirmd** [reply]
Может ещё гранатомёт добавим муниципалам? Или Бин беги? 🤭
